users/arxiv/users/legacy/accounts.py

```diff
@@ -13,6 +13,7 @@
 from arxiv.base import logging
 from .. import domain
 from . import util, endorsements, exceptions, models
+from .passwords import hash_password
```

Comment:
Seems like hash_password lives in a few different places. Above "from arxiv.users.legacy.password import hash_password". Are these different?

Reply:
It should only be in users/arxiv/users/legacy/passwords.py
Let me know if you find code where that is duplicated.

Reply:
I see that it is duplicated in `users/arxiv/users/legacy/cookies.py`